libr/core/panels.c

```diff
@@ -1588,11 +1588,14 @@ static void addPanelFrame(RCore *core, const char *title, const char *cmd, const
 		if (cmd) {
 			p->cmd = r_str_newf (cmd);
 		} else {
-			p->cmd = "";
+			p->cmd = r_str_newf ("");
```

Comment:
just use strdup(), no need to use a wrapper format functoin that does va_arg.. for just "" :P

Reply:
yeah in fact, just r_str_new should be better 'cuz that's wrapping strdup internally plus take care of if (!str) case. in any way, r_str_newf is totally unnecessary here and i certainly replace them all thanks :)